src/corelibs/U2View/src/ov_sequence/GSequenceLineViewAnnotated.cpp

```diff
@@ -246,7 +245,7 @@ void GSequenceLineViewAnnotated::mouseDoubleClickEvent(QMouseEvent *me) {
     }
     const QVector<U2Region> annotationRegions = annotation->getRegions();
     foreach (const U2Region &region, annotationRegions) {
-        CHECK_CONTINUE(region.contains(lastPressPos));
+        CHECK_CONTINUE(region.contains(renderArea->coordToPos(renderAreaPoint)));
```

Comment:
nit: coordToPos can be moved out of the loop into a separate variable to simplify debugging

Reply:
OK